site/about/style-guide.qmd

```diff
@@ -0,0 +1,230 @@
+
+### Focus on teamwork
+Keep your audience involved in the process. Engage the reader with questions, and appeal to their interests. 
+
+| Correct | Incorrect | 
+|------|-----|
+|   What if you were able to more easily, more effortlessly discern uncertainty, unleashing the full potential of your model outputs? That’s the power of ValidMind.  |  ValidMind easily and effortlessly discerns uncertainty, unleashing the full potential of model outputs.  |
+: Audience engagement examples {.hover}
+
+# Conventions
+A message can only make an impact when it is received, so presentation is (almost) everything.
+
+## Comprehensive assistance
+Conveying knowledge and empowering users starts in the product interface. While external documentation can be helpful for supporting advanced functionality, users should not be lost on the basics without further reading. 
+
+- **Make it interactive.** Embedded user tutorials should exist whenever possible, and be intuitive and not obstructive. 
+- **Show only what’s necessary when necessary.** [Progressive disclosure](https://en.wikipedia.org/wiki/Progressive_disclosure) can help guide workflows as well as minimize visual clutter. 
+
+## American English
+While the ValidMind community spans far and wide, its heart finds its home in Palo Alto, California. When writing for ValidMind, keep things consistent by using [American English](https://americanenglish.state.gov/) spelling and grammar conventions. 
+
+| Correct | Incorrect | 
+|------|-----|
+|   At the center of this page, you will see the elements organized from most recent to least recent by default.  |  At the centre of this page, you will see the elements organised from most recent to least recent by default.  |
+: American English examples {.hover}
+
+## Formatting
+
+### Sentence case
+In general, follow [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) to minimize the confusion of when to capitalize and when not to capitalize. 
+
+| Correct | Incorrect | 
+|------|-----|
+|   Get started with ValidMind  |  Get Started with ValidMind  |
+: Sentence case examples {.hover}
+
+### Headings
+- **Make them imperative!** Individual task headings are a call to action. Gerunds (“-ing”) are acceptable when introducing a set of instructions with individual tasks.  
+- **Avoid numbered headings.** Most headers do not require numbering, as order can be discerned from context. Numbering headers can make it difficult to shift information around when information changes.  
+- **Don’t use terminal punctuation.** While headings should be in sentence case, they are not sentences. 
+
+| Correct | Incorrect | 
+|------|-----|
+|   Login to ValidMind  |  1. Logging into ValidMind.  |
+: Heading examples {.hover}
+
+### Emphasis
+Use emphatic styling sparingly, in order not to overwhelm the reader with visual distractions.
+
+- **Bolding:** Some light bolding can be helpful to draw attention to core concepts. 
```

Comment:
Not sure how to integrate this best, but we also use bolding to highlight UI elements you interact with. 

E.g. `2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).`

Reply:
Can you add the following suggestion? Not sure I can add it on my own pull request (actually, let me try now...)...

```
- **Bolding:** Some light bolding can be helpful to draw attention to core concepts. Bolding is also used to highlight UI elements that the user can interact with.
```

Reply:
Done, added a suggestion.

Reply:
Oops, I went and added my own out of curiosity to see if it would work... (turns out it does!)